File: src/main/java/com/lawencon/elearn/dto/login/LoginResDto.java
package com.lawencon.elearn.dto.login;

public class LoginResDto {
	private Long id;
	private String roleCode;
	private String token;
	private String fullname;
	public Long getId() {
		return id;
	}
	public void setId(Long id) {
		this.id = id;
	}
	public String getRoleCode() {
		return roleCode;
	}
	public void setRoleCode(String roleCode) {
		this.roleCode = roleCode;
	}
	public String getToken() {
		return token;
	}
	public void setToken(String token) {
		this.token = token;
	}
	public String getFullname() {
		return fullname;
	}
	public void setFullname(String fullname) {
		this.fullname = fullname;
	}
}

File: frontend/src/app/dto/login/login.res.dto.ts
export class LoginResDto{
    private _id! : number
    private _roleCode! : string
	private _token! : string
	private _fullname! : string

    public get id(): number {
        return this._id;
    }

    public set id(id: number) {
        this._id = id;
    }

    public get roleCode(): string {
        return this._roleCode;
    }

    public set roleCode(roleCode: string) {
        this._roleCode = roleCode;
    }

    public get token(): string {
        return this._token;
    }

    public set token(token: string) {
        this._token = token;
    }

    public get fullname(): string {
        return this._fullname;
    }

    public set fullname(fullname: string) {
        this._fullname = fullname;
    }

    
}